src/app/content/components/TableOfContents/styled/wrapper.tsx

```diff
@@ -58,6 +59,9 @@ export const SidebarBody = styled.div<{isOpen: State['tocOpen']}>`
     margin-top: -${toolbarMobileHeight}rem;
     top: ${bookBannerMobileMiniHeight}rem;
     height: calc(100vh - ${navMobileHeight + bookBannerMobileMiniHeight}rem);
+    max-height: calc(100vh - ${bookBannerMobileMiniHeight}rem);
+    /* bottom padding for scrolling on iphone */
+    padding-bottom: ${bottomPadding}rem;
```

Comment:
you want this padding at the bottom of the scrollable element, which i think is the `ul`, so that the padding is only visible when you scroll it down